src/pyOpenMS/pxds/IMSAlphabet.pxd

```diff
@@ -14,25 +14,46 @@ cdef extern from "<OpenMS/CHEMISTRY/MASSDECOMPOSITION/IMS/IMSAlphabet.h>" namesp
     
     cdef cppclass IMSAlphabet "OpenMS::ims::IMSAlphabet":
         IMSAlphabet() nogil except +
+            # wrap-doc:
+                #   Holds an indexed list of bio-chemical elements.
+                #   -----
+                #   Presents an indexed list of bio-chemical elements of type (or derived from
+                #   type) 'Element'. Due to indexed structure 'Alphabet' can be used similar
+                #   to @c std::vector, for example to add a new element to 'Alphabet' function
```

Comment:
best to remove "@c" similar to @p, @?, they do not reference to anything in the DocString.